support-frontend/assets/helpers/paymentIntegrations/oneOffContributions.js

```diff
@@ -64,6 +64,7 @@ export type StripeChargeData = {|
     stripePaymentMethod: StripePaymentMethod,
   },
   acquisitionData: PaymentAPIAcquisitionData,
+  publicKey: string
```

Comment:
This tells the Payment API which backend Stripe account it should use to process the card token, in order to remain consistent with the front-end decision making (via the setting switch) about which account to use.

Reply:
Got it- I thought it must be something like this